packages/stark-core/src/modules/routing/services/routing.service.ts

```diff
@@ -235,7 +236,7 @@ export class StarkRoutingServiceImpl implements StarkRoutingService {
 		options?: HookRegOptions
 	): Function {
 		// FIXME: this tslint disable flag is due to a bug in 'no-useless-cast' rule (https://github.com/SonarSource/SonarTS/issues/650). Remove it once it is solved
-		/* tslint:disable:no-useless-cast */
+		/* tslint:disable no-useless-cast */
```

Comment:
The correct format is: `/* tslint:disable:no-useless-cast */` as in the file above otherwise all rules are disabled ;)